### DISCUSSION TITLE
Respiratory..

### DISCUSSION THREAD

**Discussion by TimAshe77AI** [community/community#135840] [Repositories] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Projects

### Body

I own few some enterprise others completed this profile an the 2 here are just practice one im curious as why everyone got a problem with my decision to stop my enterprise projects ..there I said it what's the problem

**Comment by TimAshe77AI** [comment]
It's called my 1st project  in enterprise but I see another static the comments lot of opinions I guess whoever wants to answer  feel free

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @TimAshe77AI ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category and sharing more details. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by TimAshe77AI** [reply, marked as answer]
Hey do you know this person this TimAsheAI
